### DISCUSSION TITLE
Could I get some help with tShock.

### DISCUSSION THREAD

**Discussion by Inferno0701** [Pryaxis/TShock#1733] [General]
I don't know if this is a bug or if its because I'm using a lower end device, but I as soon as i boot the Terraria 1.3.0.7 mobile tShock server on version 4.3.8.0, the server crashes after a very short delay and I'm left with an error log.

[Server API] Error Server crashed due to an unhandled exception:
System.ArgumentNullException: The value cannot be null.
Name of parameter : path
   à System.IO.Path.GetFullPathInternal(String path)
   à System.IO.File.InternalGetLastWriteTimeUtc(String path, Boolean checkHost)
   à System.IO.File.GetLastWriteTime(String path)
   à Terraria.Main.DedServ()
   à Terraria.ProgramServer.InnerStart(String[] args)

Screenshot: (https://user-images.githubusercontent.com/65196994/81616929-6e2b1380-93b2-11ea-9bce-cea58705d21e.png)

I really hope you can fix this, I'm very excited in upgrading my regular server into a tShock plugin capable server.

**Comment by hakusaro** [comment]
Do you have a `ServerLog.txt`? Other log files in and out of the tshock folder? Directory structure (file and folder layout) of the TShock server? What files are in which folders?

Please note that #1721 addressees the fact that we don't really support mobile because it's an old version of the game with exploits and stuff, so we're hoping 1.4 will normalize everything.

**Comment by Inferno0701** [comment]
Oh, I thought this was a mobile remake of tShock. Thanks for the info.

**Comment by hakusaro** [comment]
We’re working on it! As soon as mobile updates for 1.4 we will definitely work to get it supported.

Lucas Nicodemus
http://keybase.io/xn

**Comment by Inferno0701** [comment]
Alright, cant wait to try it out. I'm gonna show my friends some cool magic
tricks.
